pkg/meta/annotations.go

```diff
@@ -123,7 +123,16 @@ func IsPauseAnnotationSet(annotations map[string]string) bool {
 // UseVClusterOps returns true if all admin commands should use the vclusterOps
 // library rather than admintools.
 func UseVClusterOps(annotations map[string]string) bool {
-	return lookupBoolAnnotation(annotations, VClusterOpsAnnotation)
```

Comment:
Can we just change `lookupBoolAnnotation` so that it accepts an argument to use for the default value?